scripts/ansible/oe-windows-acc-setup.yml

```diff
@@ -17,9 +17,7 @@
         seven_zip_url:       "https://oejenkins.blob.core.windows.net/oejenkins/7z1806-x64.msi"
         seven_zip_hash:      "F00E1588ED54DDF633D8652EB89D0A8F95BD80CCCFC3EED362D81927BEC05AA5"
         vs_buildtools:       "https://oejenkins.blob.core.windows.net/oejenkins/vs_buildtools_2017.exe"
-        vs_buildtools_hash:  "6F49872B04A0EAEDF5ED96AB25F7697062D81A419D45D9970E41784F31165BF2"
-        node_url:            "https://oejenkins.blob.core.windows.net/oejenkins/node-v10.16.3-x64.msi"
-        node_hash:           "f68b75eea46232adb8fd38126c977dc244166d29e7c6cd2df930b460c38590a9"
+        vs_buildtools_hash:  "9822B4C851E14D9658BABD1533F66F518C6169196E985FE5713B2774128832AE"
```

Comment:
How was this working before with incorrect hash?

Reply:
If the cached version of this package is also updated on the backend, have we considered, caching multiple versions that can be referenced via different URLs (e.g. `https://oejenkins.blob.core/windows.net/oejenkins/v3/vs_buildtools_2017.exe`)? It seems brittle today that the update and merge of the oe-windows-acc-setup.yml needs to be synchronized with updates of the cached copies of the targets.